src/main/java/uk/gov/crowncommercial/dts/scale/cat/model/agreements/ProjectEventType.java

```diff
@@ -6,4 +6,8 @@
 public class ProjectEventType {
 
     private String type;
+
+    private String description;
+
+    private  Boolean preMarketActivity;
```

Comment:
Formatting looks off again - everything is indented 4 spaces where it should be 2 and there are 2 spaces between `private  Boolean preMarketActivity`